src/components/Radio/index.tsx

```diff
@@ -56,15 +56,28 @@ const StyledRadio = styled(ReakitRadio)`
   opacity: 0.01;
 `
 
-const Radio = ({
+type RadioProps = {
+  checked?: boolean
+  children: React.ReactNode
+  disabled?: boolean
+  name: string
+  onBlur?(...args: unknown[]): unknown
```

Comment:
handlers/name/value/disabled/checked are not needed as they are in InputHTMLAttributes<HTMLInputElement>